### ISSUE TITLE
multiple requests never worked

### ISSUE THREAD

**Issue by cltj** [microsoftgraph/microsoft-graph-docs-contrib#2901]
I have tried this multiple times throughout the year and not now nor ever did this request work. 
This goes for all "revoke licence" requests and "delete app" requests as well. 
Please refrain from putting up documentation that is not accurate. 

Last time i spoke to someone that handled my issue, they said they will remove it but nothing happend. 
Please check the requests you put out and only once it it working put up the documentation for it. 

In advance thank you. 

Regards, 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4017c1d1-7a37-4ade-10d4-09e589c61101
* Version Independent ID: 44940a3c-18d3-7db9-3a20-98bb69cc52f1
* Content: [Delete iosVppAppAssignedLicense - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/intune-apps-iosvppappassignedlicense-delete?view=graph-rest-beta)
* Content Source: [api-reference/beta/api/intune-apps-iosvppappassignedlicense-delete.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/intune-apps-iosvppappassignedlicense-delete.md)
* Product: **intune**
* Technology: **microsoft-graph**
* GitHub Login: @dougeby
* Microsoft Alias: **MSGraphDocsVteam**
